newIDE/app/src/Debugger/index.js

```diff
@@ -188,6 +215,23 @@ export default class Debugger extends React.Component<Props, State> {
       }));
     } else if (data.command === 'hotReloader.logs') {
       // Nothing to do.
+    } else if (data.command === 'console.log') {
+      // Filter out unavoidable warnings that do not concern non-engine devs.
+      if (
+        data.payload.group === 'JavaScript' &&
+        (data.payload.message.includes('Electron Security Warning') ||
+          data.payload.message.includes(
+            'Warning: This is a browser-targeted Firebase bundle'
+          ))
```

Comment:
Nitpicking: might be worth extracting this into a function so that it's easy to enhance with other filters in the future.